packages/app-shortcut/ios/Plugin/AppShortcutPlugin.swift

```diff
@@ -7,12 +7,64 @@ import Capacitor
  */
 @objc(AppShortcutPlugin)
 public class AppShortcutPlugin: CAPPlugin {
+    public let tag = "AppShortcuts"
+    public static let onAppShortcutEvent = "onAppShortcut"
+    public static let userInfoShortcutItemKey = "shortcutItem"
     private let implementation = AppShortcut()
 
-    @objc func echo(_ call: CAPPluginCall) {
-        let value = call.getString("value") ?? ""
-        call.resolve([
-            "value": implementation.echo(value)
+    override public init() {
+        super.init()
+        NotificationCenter.default.addObserver(self, selector: #selector(self.handleAppShortcutNotification), name: NSNotification.Name(AppShortcutPlugin.onAppShortcutEvent), object: nil)
+    }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self)
+    }
+
+    @objc func get(_ call: CAPPluginCall) {
+        implementation.get(completion: { result in
+            call.resolve(result.toJSObject() as! JSObject)
```

Comment:
```suggestion
            if let result = result?.toJSObject() as? JSObject {
                call.resolve(result)
            }
```

Please try to avoid forced type conversions.